app/controllers/api/internal/v2/laa_references_controller.rb

```diff
@@ -47,6 +50,14 @@ def enqueue_unlink
           )
         end
 
+        def check_defendant_presence!
+          laa_reference = LaaReference.find_by(defendant_id: transformed_params[:defendant_id])
```

Comment:
`find_by` does **not** trigger an exception

`find_by1` does, but we want to customise the message (line #57), so we don't use it.